tests/scripts/perf-slices.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Yes!  Exactly the kind of thing I was thinking of.  You can even imaging having specific run commands or args to be passed in for each of these - for example, to handle the perflab case that's different than these.